docs/en/writing-running-appium/ios/audio-capture.md

```diff
@@ -0,0 +1,85 @@
+
+
+### Client Side API
+
+Since Appium 1.18.0 there is a possibility to record iOS audio stream and save
+it to a file, which could be then retrieved on the client side.
+Apple does not provide any API to directly retrieve the audio stream from a
+Simulator or a real device, but it is possible to redirect that stream to
+the host machine, where it could be captured.
+
+#### mobile: startAudioRecording
+
+Starts audio recording process on the given host audio input with the given
+audio compression parameters.
+
+#### Supported arguments
+
+ * _audioInput_: The name of the corresponding audio input device to use for the capture, for example `:1`. The full list of capture devices could be shown using `ffmpeg -f avfoundation -list_devices true -i ""` Terminal command. This argument is mandatory.
+ * _audioCodec_: The name of the audio codec. `aac` is used by default.
+ * _audioBitrate_: The bitrate of the resulting audio stream. `128k` by default.
+ * _audioChannels_: The count of audio channels in the resulting stream. Setting it to `1` will create a single channel (mono) audio stream. `2` by default.
+ * _audioRate_: The sampling rate of the resulting audio stream. `44100` by default.
+ * _timeLimit_: The maximum recording time, in seconds. The default value is `180`, the maximum value is `43200` (12 hours)
+ * _forceRestart_: Whether to restart audio capture process forcefully when startRecordingAudio is called (`true`) or ignore the call until the current audio recording is completed (`false`, the default setting).
+
+#### mobile: stopAudioRecording
+
+Stops the audio recording process previously started by `startAudioRecording` call. The result of the method call is a base64-encoded .mp4 file containing the audio content recorded since `startAudioRecording` was called. Subsequent method calls will return the same result.
+If `startAudioRecording` was not called before then an empty string is returned.
+
+#### Usage Examples
+
+```java
+// Java
+driver.executeScript("mobile: startAudioRecording", ImmutableMap.of("audioInput", ":1"));
+Thread.sleep(10000);
+byte[] mp4Data = Base64.getMimeDecoder().decode(driver.executeScript("mobile: stopAudioRecording"));
+try (FileOutputStream fos = new FileOutputStream("out.mp4")) {
+   fos.write(mp4Data);
+}
+```
+
+
+### Server Requirements
+
+It is mandatory that the host machine has [FFMPEG](https://www.ffmpeg.org/download.html) installed and available in PATH.
+
+Mac OS since version 10.15 requires applications that record Microphone audio to be explicitly enabled in System Preferences->Security & Privacy->Privacy->Microphone tab.
+Make sure either FFMPEG itself or the parent Appium process (e.g. Terminal) is present in that list.
+
+The feature is potentially unsecure, so it must be explicitly allowed
+on the server side.
+The feature name is `audio_record`.
+See https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/security.md for more details.
```

Comment:
`[Security](docs/en/writing-running-appium/security.md)` instead of https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/security.md is enough since a documentation script handles such paths.

Reply:
changed